lightly_studio_view/src/lib/components/ExportSamples/ExportSamples.svelte

```diff
@@ -17,19 +17,25 @@
     import { useExportDialog } from '$lib/hooks/useExportDialog/useExportDialog';
 
     const { isExportDialogOpen, openExportDialog, closeExportDialog } = useExportDialog();
-    let isSelectionInverted = $state(false);
 
+    let exportType = $state<'annotations' | 'samples'>('samples');
     let datasetId = page.params.dataset_id;
-    let tagIdToExport = $state('');
 
+    //
+    // Samples export
+    //
```

Comment:
Moved here all related only to samples export. Note that that's tag handling and error handling (annotations export does not call the api, just links a url).